### PR TITLE
CLOUDP-200669 Add more context on podman errors

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Tentatively getting podman error into our error

_Jira ticket:_ CLOUDP-200669

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by blva — DISMISSED** [review]
LGTM, good to get docs approval here maybe

**Review by tibulca — DISMISSED** [review]

File: internal/podman/client.go
Comment (tibulca) [inline internal/podman/client.go:335]: was this line lost after merging master? `was this line lost after merging master?
`err = fmt.Errorf("%w: you may rerun with this command with the flag '--debug' to gather more information", err)`

should this be?
```suggestion
		err = was this line lost after merging master?
`err = fmt.Errorf("%w: you may rerun with this command with the flag '--debug' to gather more information", extractErrorMessage(exitErr))`
```

**Review by blva — APPROVED** [review]

**Pull request merged**
